store/tikv/gc_worker_test.go

```diff
@@ -30,7 +30,7 @@ type testGCWorkerSuite struct {
 var _ = Suite(&testGCWorkerSuite{})
 
 func (s *testGCWorkerSuite) SetUpTest(c *C) {
-	s.store = newTestStoreWithBootstrap(c)
```

Comment:
bootstrap create domain,  which runs the ddl goroutine, and ddl visit store.
at the same time, line 34 visit it too: `s.store.oracle = s.oracle`